openpype/hosts/maya/plugins/publish/collect_animation.py

```diff
@@ -58,17 +58,30 @@ def process(self, instance):
         if instance.data.get("farm"):
             instance.data["families"].append("publish.farm")
 
-        # Collect user defined attributes.
-        if not instance.data.get("includeUserDefinedAttributes", False):
-            return
-
-        user_defined_attributes = set()
-        for node in hierarchy:
-            attrs = cmds.listAttr(node, userDefined=True) or list()
-            shapes = cmds.listRelatives(node, shapes=True) or list()
-            for shape in shapes:
-                attrs.extend(cmds.listAttr(shape, userDefined=True) or list())
+        # Backwards compatibility for attributes.
+        backwards_mapping = {
+            "write_color_sets": "writeColorSets",
+            "write_face_sets": "writeFaceSets",
+            "include_parent_hierarchy": "includeParentHierarchy",
+            "include_user_defined_attributes": "includeUserDefinedAttributes"
+        }
+        for key, value in backwards_mapping.items():
+            if key in instance.data:
```

Comment:
This is missing the logging that `collect_pointcache` does do.

Reply:
Sure.